akka-actor-typed-tests/src/test/resources/application.conf

```diff
@@ -0,0 +1,2 @@
+akka.loglevel=warning
+akka.log-dead-letters=off
```

Comment:
will this work, we create the AS with:

```
    if (config eq ActorTestKit.noConfigSet) ActorSystem(ActorTestKitGuardian.testKitGuardian, name)
    else ActorSystem(ActorTestKitGuardian.testKitGuardian, name, config)
```

In the case where config is passed then it will not load application.conf. Can be confusing that it loads it for a tests and then someone changes the test to add some config property and then not.

That should be changed in ActorTestKit anyway. I don't like loading application.conf from testkit. That's typically the application's real config. Perhaps we could define that ActorTestKit should always try to load a `application-test.conf`?

Reply:
It definitely silences all the info logging that is otherwise present, so guess all those tests using info logging are with no custom config.

Why wouldn't we want to load the real config, if not the users would have to always duplicate all application specific settings in the `application-test.conf` or else reading the settings would fail?

When I think of it that means that the classpath order decides which file who wins, so maybe a application-test.conf and fallback to application.conf would be the right sandwich here?

Reply:
you wouldn't be able to write a unit test that is not using the real application.conf (e.g. you don't want to start remoting if you are writing a test for a specific Behavior)

if someone wants to application.conf in tests they can include it from `application-test.conf`

I prefer to have all config in the test code itself (self contained), but that would mean that those two lines you added would have to be copied to all our test classes, since we decided to not have any Akka internal base class or other internal infrastructure.

Reply:
Humhum. Yeah, I see that in the other test suites we have used a `reference.conf` for test-global settings (mostly enabling/disabling serialize messages).

I think I'd prefer to do that `application-test.conf` in a separate PR/issue though, as this one is not about any features the users will see.

Reply:
Created #25708 to track that

Reply:
Reverted the application.conf now.